### PR TITLE
Update funfacts.txt

### PR DESCRIPTION
**Pull request by Zetvue**

### CODE REVIEW

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -19,3 +20,8 @@ At one point in Minecraft, there was a rare chance of Snow Golems naturally spaw
 When you're asked to update your mods, you can click on one to stop it\nfrom being replaced with SkyClient's latest version.
 Wyvest
 Forge is actually just 2 files, a jar and a json.
+Make sure to drink water! You never know if you might be dehydrated :)
+People asking you to "test their mod" are always trying to rat you. The only ones you can trust are ones widely trusted by the community.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:24]: you forgor newline :skull:

File: docs/assets/funfacts.txt [hunk 2]
```diff
@@ -19,3 +20,8 @@ At one point in Minecraft, there was a rare chance of Snow Golems naturally spaw
...
+SBE bad lol
+Make sure to read up on this! It's **essential** for mod safety.\nhttps://github.com/ILikePlayingGames/SkyblockModSafety
```
Comment (KTibow) [inline docs/assets/funfacts.txt:26]: i wonder if we need a poll for this or smth

Comment (Zetvue) [reply docs/assets/funfacts.txt:26]: hmm do we?

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:24]: uh oh

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:26]: i mean people might not bother going to smth, especially given they can't click on it

**Pull request merged**